zebra-state/src/service.rs

```diff
@@ -389,8 +389,7 @@ impl StateService {
         let full_verifier_utxo_lookahead = max_checkpoint_height
             - HeightDiff::try_from(checkpoint_verify_concurrency_limit)
                 .expect("fits in HeightDiff");
-        let full_verifier_utxo_lookahead =
-            full_verifier_utxo_lookahead.expect("unexpected negative height");
+        let full_verifier_utxo_lookahead = full_verifier_utxo_lookahead.unwrap_or(block::Height(0));
```

Comment:
Also, there is a significant amount of consensus code that isn't implemented due to the mandatory checkpoint, including:
* difficulty adjustment for heights less than 17
* slow start block subsidy
* activating NU5 or NU6 at genesis or block 1 (which allows transactions v4,v5,v6)
* contextual validation of the genesis block (it has to be a checkpoint)

If the eventual goal of this PR series is to make all those things work, then this list should become a tracking issue. Until we do all those things, Zebra won't be compatible with `zcashd` in an alternate testnet.

However, if the eventual goal is to test ZSAs, then forking the current testnet chain by adding NU6 at a nearby height might be more reliable (and much faster to implement, although slower to sync).